Documentation Standard Updates (#27)
fivetran/dbt_pendo: README.md

</p>

# Pendo Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_pendo/))
## What does this dbt package do?
- Produces modeled tables that pendoage Pendo data from [Fivetran's connector](https://fivetran.com/docs/applications/pendo) in the format described by [this ERD](https://fivetran.com/docs/applications/pendo#schemainformation) and builds off the output of our [Pendo source package](https://github.com/fivetran/dbt_pendo_source).

- Enables you to understand how users are experiencing and adopting your product. It achieves thi by:

> - Calculating usage of features, pages, guides, and the overall product at the account and individual visitor level
> - Enhancing event stream tables with visitor and product information, referring pages, and features to track the customer journey through the application
> - Creating daily activity timelines for features, pages, and guides that reflect their adoption rates, discoverability, and usage promotion efficacy
> - Directly tying visitors and features together to determine activation rates, power-usage, and churn risk

Author: ```suggestion
  - Calculating usage of features, pages, guides, and the overall product at the account and individual visitor level
  - Enhancing event stream tables with visitor and product information, referring pages, and features to track the customer journey through the application
  - Creating daily activity timelines for features, pages, and guides that reflect their adoption rates, discoverability, and usage promotion efficacy
  - Directly tying visitors and features together to determine activation rates, power-usage, and churn risk
```

> The following table provides a detailed list of all models materialized within this package by default.
> > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_pendo/#!/overview?g_v=1).
> 
> | **Model**                | **Description**                                                                                                                                |

Author: ```suggestion

<!--section=“pendo_transformation_model"-->

The following table provides a detailed list of all tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_pendo/#!/overview?g_v=1).

| **Table**                | **Description**                                                                                                                                |
```

| [pendo__guide_event](https://fivetran.github.io/dbt_pendo/#!/model/model.pendo.pendo__guide_daily_metrics)             | The event stream of different kinds of interactions visitors have with guides. Enriched with any visitor and/or account passthrough columns, as well as the application and platform that the event occurred on. |
| [pendo__visitor_feature](https://fivetran.github.io/dbt_pendo/#!/model/model.pendo.pendo__visitor_feature)             | Each record represents a unique combination of visitors and features, aimed at making "power-users" of particular features easy to find. Includes metrics reflecting the longevity and frequency of feature usage. |

> 

Author: ```suggestion

<!--section-end-->

```
